node/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# Miden node Dockerfile
+
+# Setup image builder
+FROM rust:1.76-slim-bookworm AS builder
+
+# Set labels
+LABEL org.opencontainers.image.authors=miden@polygon.io \
+      org.opencontainers.image.url=https://0xpolygonmiden.github.io/ \
+      org.opencontainers.image.documentation=https://github.com/0xPolygonMiden/miden-node \
+      org.opencontainers.image.source=https://github.com/0xPolygonMiden/miden-node \
+      org.opencontainers.image.vendor=Polygon \
+      org.opencontainers.image.licenses=MIT
+
+ARG CREATED
+ARG VERSION
+ARG COMMIT
+LABEL org.opencontainers.image.created=$CREATED \
+      org.opencontainers.image.version=$VERSION \
+      org.opencontainers.image.revision=$COMMIT
```

Comment:
I don't think we need the labels for the builder image. The image that will be published is the image with the node. Please move it down to the other one, and move the `ARG` down as further down as possible.